docs/controllers/services/examples/http-nginx-with-healthcheck-numeric-values.yml

```diff
@@ -11,15 +11,16 @@ metadata:
 spec:
   type: LoadBalancer
   selector:
-    app: nginx-example
+    matchLabels:
```

Comment:
You modified the service object here and elsewhere, which I believe does not support [set-based requirements](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements) today.